README.md

```diff
@@ -112,7 +112,7 @@ If you have questions or comments, please create an issue.
 
 > Mozilla pledges for a healthy internet and with it, has opportunities to contribute to its open-source projects.
 
-- [Good First Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=good-first-bug) - bugs that developers have identified as a good introduction to the project.
+- [Good First Bugs](https://codetribute.mozilla.org/projects/devtools?tag%3Dgood-first-bug) - bugs that developers have identified as a good introduction to the project.
```

Comment:
Thanks for the suggestion! I took a look at the link and it seems to be just for their devtools project. However, the site as a whole to look for contributions is great. So I'd suggest to reverting the change you had, but just simply add in the site as a whole underneath.

```suggestion
- [Good First Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=good-first-bug) - bugs that developers have identified as a good introduction to the project.
- [Codetribute](https://codetribute.mozilla.org/) - find your first code contribution with Mozilla 
```